app/post/modify/post-toolbox.directive.js

```diff
@@ -0,0 +1,78 @@
+module.exports = PostToolboxDirective;
+
+PostToolboxDirective.$inject = [
+    'PostStatusService',
+    'PostMetadataService',
+    'moment',
+    '$rootScope'
+];
+function PostToolboxDirective(
+    PostStatusService,
+    PostMetadataService,
+    moment,
+    $rootScope
+) {
+    return {
+        restrict: 'E',
+        scope: {
+            post:  '='
+        },
+        templateUrl: 'templates/posts/modify/post-toolbox.html',
+        link: function ($scope) {
+            $scope.source = PostMetadataService.formatSource($scope.post.source);
+            $scope.post.user = PostMetadataService.loadUser($scope.post);
+
+            $scope.statuses = PostStatusService.getStatuses();
+
+            formatDates();
+
+            $scope.changeStatus = function (status) {
```

Comment:
Can you use the newer code style for this directive?
ie. named functions, bind to $scope at the top of `link`, move link to a named function.